### DISCUSSION TITLE
Style flicker

### DISCUSSION THREAD

**Discussion by wilfredjonathanjames** [iddan/react-spreadsheet#407] [Ideas]
CSS seems to be injected well after page load. Using NextJS and React latest, I'm seeing a [significant style flicker](https://youtu.be/eAaeczzNqYs).

I would really rather overload all of the styles myself, as I'm going to be modifying these heavily. Can I suggest something like [`react-select`'s `unstyled` prop](https://react-select.com/styles#the-unstyled-prop) but without style flicker?

Some clear examples of how to restyle this would also be appreciated.

**Comment by iddan** [comment]
Hey, @wilfredjonathanjames. For now I don't think I would take react select's approach. But I'm open for suggestions on how to improve the style flicker in NextJs
